clusterloader/cluster_loader.go

```diff
@@ -156,32 +158,16 @@ func createTemplate(baseName string, ns *v1.Namespace, configPath string, numObj
 		if tuning != nil {
 			if tuning.Templates.RateLimit.Delay != 0 {
 				framework.Logf("Sleeping %d ms between template creation.", tuning.Templates.RateLimit.Delay)
-				time.Sleep(time.Duration(tuning.Templates.RateLimit.Delay) * time.Millisecond)
+				time.Sleep(tuning.Templates.RateLimit.Delay * time.Millisecond)
```

Comment:
FYI - in k8s components we are using this flags.DurationVar(...) which supports it.